src/Modules/addAndremove.js

```diff
@@ -0,0 +1,74 @@
+const demo = () => {
+  let value = 0;
+  const array = [];
+  const btnArray = [];
+  let i = 0;
+  let id = 10000;
+  const inputHolder = document.querySelector('.input-holder');
+  inputHolder.addEventListener('keyup', (event) => {
+    const li = document.createElement('li');
+    li.className = 'to-do-list-item';
+    const ulElement = document.querySelector('.li-item-container');
+    if (event.keyCode === 13 && inputHolder.value.trim().length > 0) {
+      const obj = {
+        description: inputHolder.value,
+        complete: false,
+        index: (value += 1),
+      };
```

Comment:
Kindly check the index of the tasks after saving them to local storage as it starts from `1` after every reload of the page. 🔄 
This causes a bug in the app that replaces the existing tasks that have the same index as the newly created tasks. 

- [ ] Kindly make sure that each task has a unique index to fix the bug mentioned above. ♻️